src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.icon_small {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
```

Comment:
move it to the `html` tag

Reply:
"Remember that inputs and other interactive elements don’t inherit font styles by default."
Readme.md

Reply:
`font-family: inherit` ?